aiida/manage/configuration/migrations/migrations.py

```diff
@@ -291,7 +291,7 @@ def upgrade(self, config: ConfigType) -> None:
     def downgrade(self, config: ConfigType) -> None:
         for profile_name, profile in config.get('profiles', {}).items():
             if '_v6_backend' in profile.get('storage', {}):
-                profile.setdefault('storage', {})['backend'] = profile.pop('_v6_backend')
+                profile.setdefault('storage', {})['backend'] = profile['storage'].pop('_v6_backend')
```

Comment:
Thanks @kjappelbaum . I just checked and there is no test for this downgrade, that's why this went unnoticed. Could you please add the following test on line 93 of `tests/manage/configuration/test_migrations.py`?
```python

def test_merge_storage_backends_downgrade_profile(empty_config, profile_factory, caplog):
    """Test the downgrade of schema version 7.

    Test specifically the case that the ``storage._v6_backend`` key does not exist.
    """
    config = empty_config
    profile_a = profile_factory('profile_a', test_profile=False)
    profile_b = profile_factory('profile_b', test_profile=False)

    profile_a._attributes[profile_a.KEY_STORAGE]['_v6_backend'] = 'django'  # pylint: disable=protected-access

    config.add_profile(profile_a)
    config.add_profile(profile_b)

    config_migrated = downgrade_config(config.dictionary, 6)
    assert list(config_migrated['profiles'].keys()) == ['profile_a', 'profile_b']
    assert f'profile {profile_b.name!r} had no expected "storage._v6_backend" key' in caplog.records[0].message
```

Reply:
Thanks! TBH, i didn't have the time to check how you set up the test for this part of the code - so many thanks with helping with the full snippet!

Reply:
Sure thing, adding tests is often time consuming if you're not fully familiar and didn't want to burden you nor block this PR :) Thanks a lot for the fix!